.circleci/test-deploy.yml

```diff
@@ -25,11 +25,13 @@ workflows:
           type: approval
           requires:
             - sym/request
+          filters: *filters
```

Comment:
This `filters` block is required for these jobs to run when you cut a new release